### PR TITLE
Pdpinch/maple cherry pick 715 Removed course header from Learning MFE

### PR DESCRIPTION
**Pull request by pdpinch**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pdpinch! I've created [OSPR-6325](https://openedx.atlassian.net/browse/OSPR-6325) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: package.json [hunk 1]
```diff
@@ -33,10 +33,11 @@
...
     "@edx/frontend-component-footer": "10.1.6",
-    "@edx/frontend-enterprise-utils": "1.1.0",
-    "@edx/frontend-lib-special-exams": "1.13.3",
-    "@edx/frontend-platform": "1.12.7",
-    "@edx/paragon": "16.14.9",
+    "@edx/frontend-enterprise-utils": "1.1.1",
+    "@edx/frontend-lib-special-exams": "1.14.1",
+    "@edx/frontend-platform": "1.14.3",
+    "@edx/paragon": "16.19.0",
+    "@edx/frontend-component-header": "^2.4.3",
```
Comment (davidjoy) [inline package.json:40]: Nit: Remove the `^` here

**Review by davidjoy — APPROVED** [review]

**Comment by regisb** [comment]
@edx-webhook merge

**Comment by regisb** [comment]
@edx-community-bot merge

**Comment by openedx-webhooks** [comment]
@pdpinch 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/frontend-app-learning&entry.867055334=https://github.com/edx/frontend-app-learning/pull/781&entry.1484655318=https://github.com/pdpinch&entry.752974735=2021-12-20+15:45&entry.1917517419=2021-12-20+16:55&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by natabene** [comment]
Thank you all.

**Pull request merged**